adam-python/bdgenomics/adam/rdd.py

```diff
@@ -42,6 +42,18 @@ def __init__(self, jvmRdd, sc):
         self._jvmRdd = jvmRdd
         self.sc = sc
 
+    def cache(self):
+        self._jvmRdd.cache()
+
+    def persist(self, sl):
```

Comment:
1. `sl` should be of type [`StorageLevel`](http://spark.apache.org/docs/2.2.0/api/python/pyspark.html?highlight=storagelevel#pyspark.StorageLevel)
2. Use the Java [StorageLevels.create](https://spark.apache.org/docs/2.2.0/api/java/org/apache/spark/api/java/StorageLevels.html#create-boolean-boolean-boolean-boolean-int-) method
3. Voila:

```
self._jvmRdd.persist(self.sc._jvm.org.apache.spark.api.StorageLevels.create(sl.useDisk, sl.useMemory, sl.useOffHeap, sl.useDeserialized, sl.replication)
```

Py4j lines Python types up with Java types, so you need to line Python `bool` up with Java `boolean`, not `Scala.Boolean`, Python `int` up with Java `int`, not `Scala.Integer`, etc.